ambry-rest/src/main/java/com.github.ambry.rest/NettyMetrics.java

```diff
@@ -237,5 +247,24 @@ public NettyMetrics(MetricRegistry metricRegistry) {
     // NettyServer
     nettyServerShutdownTimeInMs = metricRegistry.histogram(MetricRegistry.name(NettyServer.class, "ShutdownTimeInMs"));
     nettyServerStartTimeInMs = metricRegistry.histogram(MetricRegistry.name(NettyServer.class, "StartTimeInMs"));
+    // ConnectionStatsHandler
+    connectionsConnectedCount =
```

Comment:
Why do you think ConnectionStatsHandler should own this? In that case, we can also argue for health check metrics, public access log metrics and so on. I feel we can have all the metrics in one place (NettyMetrics)